article/background.tex

```diff
@@ -80,13 +81,31 @@ \subsection{Software Dependencies}
-Their relevance to open science consists in providing end-users with an accessible environment, which can be ascertained to provide the requirements of a certain top-level workflow, and which does not interfere with their parent environment.
+1. VMs skip the most difficult step: Installation and coordinating dependencies is often challenging, time consuming, and requires domain knowledge reproducers and reviewers are likely to lack.
+1. Guests are self-contained and isolated from the host, which eliminates the posibility of polluting the host environment.
+1. Admins are able to safely allow mostly unrestricted usage from semi-trusted users.
+1. Package managers are imperfect, and none can be relied upon to continue to offer all legacy releases, but VMs preserve a working copy.
 
-Two of the most prominent container standards are offered by the Open Container Initiative (OCI) and Singularity.
-The former is used by applications such as Docker and Podman, and the latter by the eponymous Singularity application, which differentiates itself from OCI by not requiring root access, and thus being arguably better suited for scientific computing in high-performance computing environments.
+System virtualization offers a way to portably freeze and preserve environments, but are limited due to the size of the "full disk images".
+Additionally VM's must be be "booted" which can be costly if many instances are needed.
+Modern advances in container technology have allowed similar benefits but strip redundancy by making limited use of the host machine, specifically the hypervisor.
+Containers enable a complete working environment as small as a few Megabytes, and can be started as quickly as a normal process.
+Many container images are publicly available via public image repositories.
 
-While the reference OPFVTA article does not leverage this technology, containers can improve its portability, as well as provide a snapshot of its functioning at a certain point in time — mitigating process fragility in view of incrementing software dependency versions.
+Containers technology is not a recent invention, but the term "container" gained popularity alongside the Docker toolset.
+Over time Docker and other organizations have come together under a Linux Foundation project, the "Open Container Initiative" (OCI).
+The OCI governing body has produced an open specification for containers, which can be used by various container runtimes and toolsets.
+OCI complient container images in most cases can be executed identically with Docker, Podman, or other OCI compliant tools.
+
+While OCI images are nearly ubiquitous in the private sector, Apptainer (formerly Singularity) is a toolset that was developed specifically for High Performance Computing.
+Apptainer has support for converting OCI images into apptainer images, and has also added support to natively (TODO make sure this is true, asmacdo) run OCI containers.
```

Comment:
Please turn it into a sentence or put the TODO in a comment (LaTeX comment char is `%`).

Reply:
asmacdo's personal style guide re: todos

No new TODOS should merge unless there is an associated issue in the tracker.

So read TODO as something I intend to fix before merge.